packages/react/src/studio/getOverridesFromVariants.ts

```diff
@@ -30,8 +30,10 @@ export function getOverridesFromVariants<T>(
   const variantValuesFromProps: VariantValues = Object.keys(props)
     .filter((i) => variantValueKeys.includes(i) && props[i])
     .reduce((acc, key) => {
-      acc[key] = props[key];
-      return acc;
+      return {
```

Comment:
Is there a test on this change?

Reply:
There are tests already for this functionality. This change is purely to make TS happy because it likes object spreading, but not arbitrary string index accessing for some reason. AFAIK the previous code and this code are equivalent.